packages/node_modules/@webex/http-core/test/integration/spec/interceptor.js

```diff
@@ -22,7 +22,16 @@ describe('http-core', () => {
       })));
 
     describe('logOptions', () => {
-      it('calls logger plugin', () => {
+      it('calls logger plugin if enabled', () => {
+        // this if statement catches and skips test if verbose network logging
+        // is not enabled.
+        if (!process.env.ENABLE_VERBOSE_NETWORK_LOGGING) {
```

Comment:
Like @taymoork2 suggested, should we set the logging to be true manually to validate the test?